lib/wovnrb/services/time.rb

```diff
@@ -0,0 +1,14 @@
+module Wovnrb
+  # Provides utilities related to Time
+  class TimeUtil
+    class << self
+      def round_down_time(time, unit)
+        time - (time % unit)
+      end
+
+      def time_proc
+        -> { return Time.now.utc.sec }
+      end
```

Comment:
the naming is vague as to what it returns. This would be `current_utc_time_secs` or something. I feel like you could just return `utc_now` and let the caller convert it to secs